web/src/components/users/UserEditForm.vue

```diff
@@ -75,6 +75,9 @@
           label="Department"
           variant="outlined"
           required
+          readonly
+          disabled
```

Comment:
I think lets just drop these fields.

We don't generally use "disabled" for things that are display only, because it makes them too hard to read.
Generally, I would only use "disabled" in the case where the user edit as admin form is doing so.
i.e. a field that the user should not look at - and maybe never look at - until they do something prior, at which point the field becomes non-disabled an draws focus.